code/modules/ai/ai_behaviors/human_mobs/self_heal.dm

```diff
@@ -0,0 +1,235 @@
+	/obj/item/reagent_containers/hypospray/autoinjector/neuraline,
+	/obj/item/reagent_containers/hypospray/autoinjector/russian_red,
+	/obj/item/reagent_containers/hypospray/autoinjector/elite,
+	/obj/item/stack/medical/heal_pack/gauze/sectoid,
+	/obj/item/stack/medical/heal_pack/advanced/bruise_pack,
+	/obj/item/stack/medical/heal_pack/gauze,
+))
+
+GLOBAL_LIST_INIT(ai_burn_heal_items, list(
+	/obj/item/reagent_containers/pill/kelotane,
+	/obj/item/reagent_containers/hypospray/autoinjector/kelotane,
+	/obj/item/reagent_containers/hypospray/advanced/kelotane,
+	/obj/item/reagent_containers/hypospray/autoinjector/combat,
+	/obj/item/reagent_containers/pill/tricordrazine,
+	/obj/item/reagent_containers/hypospray/advanced/tricordrazine,
+	/obj/item/reagent_containers/hypospray/advanced/combat_advanced,
+	/obj/item/reagent_containers/hypospray/autoinjector/combat_advanced,
+	/obj/item/reagent_containers/pill/dermaline,
+	/obj/item/reagent_containers/hypospray/advanced/dermaline,
+	/obj/item/reagent_containers/hypospray/advanced/meraderm,
+	/obj/item/reagent_containers/hypospray/autoinjector/neuraline,
+	/obj/item/reagent_containers/hypospray/autoinjector/russian_red,
+	/obj/item/reagent_containers/hypospray/autoinjector/elite,
+	/obj/item/stack/medical/heal_pack/gauze/sectoid,
+	/obj/item/stack/medical/heal_pack/advanced/burn_pack,
+	/obj/item/stack/medical/heal_pack/ointment,
+))
+
+GLOBAL_LIST_INIT(ai_tox_heal_items, list(
+	/obj/item/reagent_containers/hypospray/autoinjector/antitox_mix,
+	/obj/item/reagent_containers/pill/dylovene,
+	/obj/item/reagent_containers/hypospray/advanced/dylovene,
+	/obj/item/reagent_containers/hypospray/autoinjector/dylovene,
+	/obj/item/reagent_containers/pill/tricordrazine,
+	/obj/item/reagent_containers/hypospray/advanced/tricordrazine,
+))
+
+GLOBAL_LIST_INIT(ai_oxy_heal_items, list(
+	/obj/item/reagent_containers/hypospray/autoinjector/dexalin,
+	/obj/item/reagent_containers/pill/tricordrazine,
+	/obj/item/reagent_containers/hypospray/advanced/tricordrazine,
+))
+
+GLOBAL_LIST_INIT(ai_clone_heal_items, list(
+	/obj/item/reagent_containers/hypospray/autoinjector/rezadone,
+	/obj/item/reagent_containers/hypospray/autoinjector/elite,
+))
+
+GLOBAL_LIST_INIT(ai_pain_heal_items, list(
+	/obj/item/reagent_containers/pill/tramadol,
+	/obj/item/reagent_containers/hypospray/autoinjector/tramadol,
+	/obj/item/reagent_containers/hypospray/advanced/tramadol,
+	/obj/item/reagent_containers/hypospray/autoinjector/combat,
+	/obj/item/reagent_containers/hypospray/advanced/oxycodone,
+	/obj/item/reagent_containers/hypospray/autoinjector/oxycodone,
+	/obj/item/reagent_containers/hypospray/advanced/combat_advanced,
+	/obj/item/reagent_containers/hypospray/autoinjector/combat_advanced,
+	/obj/item/reagent_containers/hypospray/autoinjector/neuraline,
+	/obj/item/reagent_containers/hypospray/autoinjector/russian_red,
+	/obj/item/reagent_containers/hypospray/autoinjector/elite,
+))
+
+GLOBAL_LIST_INIT(ai_damtype_to_heal_list, list(
```

Comment:
splinting/shrap might be interesting

Reply:
they already self splint. I will do dedicated medic stuff in a later PR.